website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
+You need to have a running Kubernetes cluster to set up PostgreSQL with DocumentDB extension on Kubernetes.
+
+You can deploy PostgreSQL with DocumentDB extension using any of our provided images.
+Please see the [Docker installation docs](../documentdb/docker.md) to learn more about the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `17-0.102.0-ferretdb-2.1.0`)
+to ensure consistency across deployments.
+For more information on the best FerretDB image to use, see the [DocumentDB release notes](https://github.com/FerretDB/documentdb/releases/).
+:::
+
+The following `postgres.yaml` file creates a PostgreSQL instance with the DocumentDB extension:
```

Comment:
This _file_ does not _create_  a PostgreSQL instance with _the_ DocumentDB extension (weird that we added _the_ there, but not before; I guess _the_ appears only after you say X four times) simply because YAML can't create anything – it is data.